src/main/scala/io/qbeast/spark/table/IndexedTable.scala

```diff
@@ -146,6 +158,8 @@ private[table] class IndexedTableImpl(
     checkColumnsToMatchSchema(latestRevision)
     // Checks if the desiredCubeSize is different from the existing one
     val isNewCubeSize = latestRevision.desiredCubeSize != qbeastOptions.cubeSize
+    // Checks if the desiredFileSize is different from the existing one
+    val isNewFileSize = latestRevision.desiredFileSize != qbeastOptions.fileSize
```

Comment:
When `fileSize` is not provided during an `append`, its value should be that of the existing revison. This is handled by the method `addRequiredParams` but at the moment it is not doing so.